brave-spark-framework-filter/src/main/java/com/github/kristofa/brave/spark/BraveSparkExceptionHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.ServerTracer;
+import spark.ExceptionHandler;
+import spark.Request;
+import spark.Response;
+import spark.utils.StringUtils;
+import zipkin.Constants;
+
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+public final class BraveSparkExceptionHandler implements ExceptionHandler {
+
+    private final Brave brave;
+
+    private final ExceptionHandler exceptionHandler;
+
+    public static BraveSparkExceptionHandler create(Brave brave, ExceptionHandler exceptionHandler) {
+        checkNotNull(brave, "brave");
+        checkNotNull(brave, "exceptionHandler");
+        return new BraveSparkExceptionHandler(brave, exceptionHandler);
+    }
+
+    private BraveSparkExceptionHandler(Brave brave, ExceptionHandler exceptionHandler) {
+        this.brave = brave;
+        this.exceptionHandler = exceptionHandler;
+    }
+
+    @Override
+    public void handle(Exception ex, Request request, Response response) {
+        try {
+            ServerTracer serverTracer = brave.serverTracer();
+            StringBuilder sb = new StringBuilder();
+            if (StringUtils.isNotEmpty(ex.getMessage())) {
+                sb.append(ex.getMessage()).append("\n");

```

Comment:
in other code, we have been only doing the exception message. probably a better idea to just follow that as it is also less code to test. People can always make fancier exception handlers

```java
      String message = throwable.getMessage();
      if (message == null) message = throwable.getClass().getSimpleName();
      span.tag(Constants.ERROR, message);
```

less code is better especially in new integrations